### PR TITLE
Add ISSUE TEMPLATES

### PR DESCRIPTION
**Pull request by Sonichigo**

### CODE REVIEW

**Comment by Sonichigo** [comment]
@krishdevdb Check it once!!
We have also gotta change repo owner in .allcontributorsrc to resetercss

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord Community
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/config.yml]: Not sure if needed, this was added when I was trying to make a discord server, but didn't go any well 😓 So Maybe remove this

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      placeholder: A detailed description of the feature request.
+      value: "I found this error in this project where ..."
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: contributing
+    attributes:
+      label: Contributing
+      description: You accept that you've read the [Contributing](https://github.com/krishdevdb/.github/blob/main/contributing.md)
```
Comment (ghost) [inline .github/ISSUE_TEMPLATE/feature_request.yml]: These links are broken now 😅

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
> @krishdevdb Check it once!!
> We have also gotta change repo owner in .allcontributorsrc to resetercss

I am thinking of abandoning all-contributors

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Sonichigo) [reply .github/ISSUE_TEMPLATE/config.yml]: Coool Cool!!

File: .github/ISSUE_TEMPLATE/feature_request.yml (hunk 2 above)
Comment (Sonichigo) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: https://github.com/krishdevdb/.github/blob/main/contributing.md this one is still valid no !👀

Comment (ghost) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: I mean we shall move them back to this repo

**Pull request closed without merging**